tests/test_transpilers_blocks.py

```diff
@@ -0,0 +1,25 @@
+from qibo import Circuit, gates
+
+from qibolab.transpilers.blocks import create_dag, initial_block_deomposition
+
+circ = Circuit(6)
+circ.add(gates.H(0))
+circ.add(gates.H(1))
+circ.add(gates.H(2))
+circ.add(gates.CZ(0, 1))
+circ.add(gates.CZ(0, 1))
+circ.add(gates.CZ(2, 3))
+circ.add(gates.CZ(1, 2))
+circ.add(gates.CZ(0, 1))
+circ.add(gates.CZ(2, 3))
+circ.add(gates.H(0))
+circ.add(gates.H(3))
+circ.add(gates.H(4))
+circ.add(gates.H(5))
+
+
+dag = create_dag(circ)
+
+blocks = initial_block_deomposition(circ)
+for block in blocks:
+    block.info()
```

Comment:
You should put them inside a test function, otherwise they are invisible to Pytest (actually worse: they would be run during module loading, so if the test were failing Pytest would not even start).

Reply:
still need to write the tests, this was just for me to check that it was working properly